templates/cloudfront.cfn.yml.j2

```diff
@@ -183,120 +176,8 @@ Resources:
-              - OPTIONS
-              - PUT
-              - PATCH
-              - POST
-              - DELETE
-            CachePolicyId: !Ref CachePolicyCMS
-            Compress: true
-            ViewerProtocolPolicy: redirect-to-https
-            LambdaFunctionAssociations:
-              - EventType: 'viewer-request'
-                LambdaFunctionARN: !Ref LambdaAtEdgeViewerRequestVersion
-          - PathPattern: /home* #The homepage
-            TargetOriginId: 'CmsOrigin'
-            AllowedMethods:
-              - GET
-              - HEAD
-              - OPTIONS
-              - PUT
-              - PATCH
-              - POST
-              - DELETE
-            CachePolicyId: !Ref CachePolicyCMS
-            Compress: true
-            ViewerProtocolPolicy: redirect-to-https
-            LambdaFunctionAssociations:
-              - EventType: 'viewer-request'
-                LambdaFunctionARN: !Ref LambdaAtEdgeViewerRequestVersion
-          - PathPattern: /article #Not sure if this page exists but just in case, a top level article page
-            TargetOriginId: 'CmsOrigin'
-            AllowedMethods:
-              - GET
-              - HEAD
-              - OPTIONS
-              - PUT
-              - PATCH
-              - POST
-              - DELETE
-            CachePolicyId: !Ref CachePolicyCMS
-            Compress: true
-            ViewerProtocolPolicy: redirect-to-https
-            LambdaFunctionAssociations:
-              - EventType: 'viewer-request'
-                LambdaFunctionARN: !Ref LambdaAtEdgeViewerRequestVersion
-          - PathPattern: /article/* #sub pages of article, also including things like /articles/blahblah/_admin
-            TargetOriginId: 'CmsOrigin'
-            AllowedMethods:
-              - GET
-              - HEAD
-              - OPTIONS
-              - PUT
-              - PATCH
-              - POST
-              - DELETE
-            CachePolicyId: !Ref CachePolicyCMS
-            Compress: true
-            ViewerProtocolPolicy: redirect-to-https
-            LambdaFunctionAssociations:
-              - EventType: 'viewer-request'
-                LambdaFunctionARN: !Ref LambdaAtEdgeViewerRequestVersion
-          - PathPattern: /logout* #SSO
-            TargetOriginId: 'CmsOrigin'
-            AllowedMethods:
-              - GET
-              - HEAD
-              - OPTIONS
-              - PUT
-              - PATCH
-              - POST
-              - DELETE
-            CachePolicyId: !Ref CachePolicyCMS
-            Compress: true
-            ViewerProtocolPolicy: redirect-to-https
-            LambdaFunctionAssociations:
-              - EventType: 'viewer-request'
-                LambdaFunctionARN: !Ref LambdaAtEdgeViewerRequestVersion
-          - PathPattern: /oauth-account-manager* #SSO
+{% for path in ['/index.*', '/_*', '/', '/home*', '/article', '/article/*', '/visualisations*', '/news-and-case-studies*', '/logout*', '/oauth-account-manager*'] %}
```

Comment:
nice, i think /article | /article/* could just be reduced to /article* (since nothing in open data would be using that)